etc/profile-m-z/retroarch.profile

```diff
@@ -0,0 +1,46 @@
+# Persistent local customizations
+include retroarch.local
+# Persistent global definitions
+include globals.local
+
+blacklist /boot
+blacklist /opt
+blacklist /usr/libexec
+blacklist /var
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+include disable-xdg.inc
+
+whitelist ${HOME}/.config/retroarch
+whitelist /usr/share/retroarch
+whitelist /usr/share/libretro
+
+include whitelist-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+
+apparmor
+caps.drop all
+netfilter
+nodvd
+nonewprivs
+notv
+nou2f
```

Comment:
`novideo`?

Reply:
I don't know what exactly `novideo` does. What are video devices?

Reply:
It seems `novideo` turns off cameras. I added it to my local copy.

Reply:
IDK how common/necessary it is. Maybe add a comment.

Reply:
Added with a comment.